massgov/mayflower-patternlab
# DP-8612 elected official contact row change, DP-8565 Mosaic Firefox Issue

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

Adds a 'column' variation to the 'contact-us' molecule.
Also fixes firefox bug for featured items in the mosaic.

## Related Issue / Ticket

- [JIRA issue  | DP-8612](https://jira.state.ma.us/browse/DP-8612)
- [JIRA issue  | DP-8565](https://jira.state.ma.us/browse/DP-8565)

## Steps to Test


1. View `http://localhost:3000/?p=pages-organization-elected-official`
2. In firefox ensure that, within mosaic under "Featured Items", the items are visible & not collapsed.
3. Ensure that the "contact us" section below "Featured Items" and above "About the..." looks as described in the ticket:  https://jira.state.ma.us/browse/DP-8612

## Screenshots

<img width="618" alt="screen shot 2018-04-19 at 12 14 40 pm" src="https://user-images.githubusercontent.com/1397914/39004428-4504b56c-43cb-11e8-98b8-11111ec02f74.png">
<img width="1240" alt="screen shot 2018-04-19 at 12 14 23 pm" src="https://user-images.githubusercontent.com/1397914/39004429-45132ce6-43cb-11e8-8be4-2474b64faff5.png">

## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
